app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -129,14 +128,14 @@ class InstalledAddonDetailsFragment : Fragment() {
                     }
                 )
             } else {
+                view.settings.isVisible = false
```

Comment:
I had to add this extra step here, as we are hiding the settings section after  `onSuccess` if the user clicks fast enough can enter to the setting page, but when the user arrives there the add-on is already disabled and the page can't be shown.

![image](https://user-images.githubusercontent.com/773158/74975490-1f925c80-53f5-11ea-8807-249b8fa602d8.png)